payment_method.go

```diff
@@ -86,3 +86,15 @@ func (c *Client) getPaymentMethodByPriority(priority PaymentMethodPriority) (out
 
 	return out, nil
 }
+
+// GetPaymentEndpointByPrimaryOrSecondary gets the payment priority based on primaryOrSecondary parameter.
+func (c *Client) GetPaymentEndpointByPrimaryOrSecondary(primaryOrSecondary PaymentMethodPriority) (out string) {
```

Comment:
We need to use this in the function `getPaymentMethodByPriority` too

Reply:
I am not exactly sure what you mean. `getPaymentMethodByPriority` is a separate helper function that has a different return type, it's being used in the `Billing` file

Reply:
There are two functions that use these enums. Let's use the helper function in both places, I only see it being used in one.

Reply:
`getPaymentMethodByPriority ` returns PaymentObject, it's being used in Billing lines 16 and 29: https://github.com/EasyPost/easypost-go/blob/a0a3342fb9ffe7028fd1e4b3924a8a9f5529012b/billing.go#L16

`getPaymentMethodEndpoint` is returning a different endpoint: "credit_cards" or "bank_accounts"

Reply:
OH. I misunderstood. This doesn't need to be a helper function then since it's only used in one place. This is a bit different than our other libs in that regard because of the enums. Please revert this change.

Reply:
No wait, it is still being used in a second place:
https://github.com/EasyPost/easypost-go/blob/b590e5b46fe0f2f0c7f40d89aecf46812f7f5f8b/referral.go#L168

Reply:
This new helper function is used in two places: `createEasypostCreditCard` and `BetaAddPaymentMethodWithPrimaryOrSecondaryAndContext`, it does not need to be reverted.

Reply:
@nwithan8 Thanks for replying while I was finishing up my lunch. Exactly, no need to revert this since the helper function is being used in two places.